optimum/bettertransformer/models/base.py

```diff
@@ -38,6 +55,10 @@ def __init__(self, config):
         self.num_heads = None
         self.embed_dim = None
         self.num_layers = None
+        self.original_layers_mapping = {}
+        # some models does not have some attributes thus needs to be ignored
+        # e.g. whisper does not have self_attn.k_proj.bias but has self_attn.v_proj.bias & self_attn.q_proj.bias
+        self.keys_to_ignore = []
```

Comment:
can't we just let the mapping empty then?

Reply:
Hmm then how would you discard a `k_proj.bias` given the sets: `q_proj`, `k_proj`, `v_proj`. Also it would break the entire logic in https://github.com/younesbelkada/optimum/blob/bt-inverse-transform-bert/optimum/bettertransformer/models/base.py#L164 so I wanted to keep things simple as these are very-corner cases